json-endpoints/jetpack/json-api-jetpack-endpoints.php

```diff
@@ -553,6 +553,7 @@
 	'render_filtered_content'  => '(int|bool=true) Set to 1 or true to render filtered content.',
 	'max_enqueue_full_sync'    => '(int|bool=false) Maximum number of rows to enqueue during each full sync process',
 	'max_queue_size_full_sync' => '(int|bool=false) Maximum queue size that full sync is allowed to use',
+	'full_sync_send_immediately' => '(bool) Skip the queue, send right away for full sync',
```

Comment:
You also need to add `full_sync_send_immediately_duration`

Reply:
https://github.com/Automattic/jetpack/pull/13963/commits/3259093487752c9e5df821e2d88ba5f76f2262c8